frontend/src/utils/artifactSort.js

```diff
@@ -0,0 +1,43 @@
+import { sortMostRecent } from './sort';
+
+export function sortByDateEdited(a, b) {
+  return sortMostRecent(a, b);
+}
+
+export function sortByDateCreated(a, b) {
+  if (a.createdAt > b.createdAt || (a.createdAt && !b.createdAt)) {
+    return -1;
+  } else if (a.createdAt < b.createdAt || (!a.createdAt && b.createdAt)) {
+    return 1;
+  }
+  return 0;
+}
+
+export function sortByName(a, b) {
+  return a.name.localeCompare(b.name);
+}
+
+export function sortByVersion(a, b) {
+  const apr_regex = /\b\d{1}\.\d{1}\.\d{1}\b/;
+  if (a.version.match(apr_regex) && b.version.match(apr_regex)) {
+    const digits_a = a.version.split('.').map(strNumeral => parseInt(strNumeral));
+    const digits_b = b.version.split('.').map(strNumeral => parseInt(strNumeral));
```

Comment:
Same comment as above, if you could change the casing for these that would be great.

Reply:
fixed